python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/monitor-server-radius.j2

```diff
@@ -0,0 +1,29 @@
+{#
```

Comment:
add the template for documentation as well

Reply:
Added